lib/client/js/ui/controls/Menu.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Insert description of Menu class
+ *
+ * @private
+ * @class
+ * @returns {Menu}
+ */
+var Menu = (function () {
+
+    Menu.prototype = new Control();
+
+    /**
+     * Decorates a jquery object with the header
+     *
+     * @private
+     * @function viewHeader
+     */
+    var viewHeader = function(jqueryObj) {
```

Comment:
Perhaps jqueryObj is too generic. I think a good name for this parameter would be `$menu` to express that it's the jquery object of the top-level menu.